tests/test_type_constraints/test_unify.py

```diff
@@ -191,3 +225,4 @@ def test_diff_callable():
     c2 = Callable[[str], str]
 
     unify_helper(c1, c2, error_msg)
+    
```

Comment:
Looks like there might be some extra whitespace here. Please remove this, and I recommend configuring your text editor to automatically trim trailing whitespace.

Reply:
Oh sorry, pep8 told me to put a new space at the end of the file. Do you want me to ignore those?

Reply:
There should be a new *line* at the end of the file, but the line shouldn't contain any spaces (watch out for indentation).